public/js/script.js

```diff
@@ -223,6 +224,62 @@ $('textarea').keyup(function(e) {
           });
     }
 
+    else if(command.split(" ")[0].trim()=="rm -r")
```

Comment:
Have you tested this? I am sure `command.split(" ")[0]` would give `rm` only in this case.

Reply:
This was a last minute change. Earlier I was doing something else which was working, I will make required changes.